docs/home/contribute/participating.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Participating in SIG-DOCS
+---
+
+{% capture overview %}
+
+SIG-DOCS is one of the [special interest groups](https://github.com/kubernetes/community/blob/master/sig-list.md) within the Kubernetes project, focused on writing, updating, and maintaining the documentation for the project as a whole.
+
+{% endcapture %}
+
+{% capture body %}
+
+Everyone is encouraged to provide content and reviews to support the Kubernetes project documentation. Anyone may open a pull request, and anyone is welcome to provide their opinions and feedback on those pull requests.
+
+Within the Kubernetes project, you may also become a member, reviewer, or approver. These levels confer additional privileges when it comes to approving and committing changes. You may also want to read [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for more details on how membership works within the Kubernetes community.
+
+## Roles and Responsibilities
+
+- Members
+
+Anyone who is a member of the Kubernetes organization can provide a review for a pull request. The SIG-DOCS team will frequently request a review for technical accuracy, and if you are a member, then you can indicate your approval by adding a comment of `/lgtm` to a pull request.
+
+- Reviewers
+
+Reviewers are individuals who have volunteered to provide their time to review documentation pull requests. You become a reviewer by adding your github handle to the [OWNERS file](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md) within the [website repo](https://github.com/kubernetes/website).
+
+Our automation will reviewers to use for reviews on pull requests, and a contributor may request your review with an `/assign [your_github_handle]` comment in the pull request. To indicate your approval of a pull request, a reviewer should add an `/approve` comment to the pull request. A reviewer may also use add a `/hold` comment on the pull request to prevent the pull request from being merged. Another reviewer or approver may also remove that hold with the comment `/hold cancel`.
+
+When a reviewer is assigned a pull request to review it is not a sole responsibility, and any other reviewer may also offer their opinions on the pull request. If a reviewer is requested, it is generally expected that the PR will be left to that reviewer to do their editorial pass on the content. For small and/or trivial pull requests, the assigned reviewer may be skipped if an approver thinks the content is sufficient and well understood. 
+
+- Approvers
+
+Approvers are the individuals with final approval over a PR for merging. An approver can indicate `/lgtm` or `/approve` to have a pull request merged, and all pull requests require at least one approver to provide their vote for the PR to be merged. In general, this is done with an `/approve` comment, expecting an `/lgtm` to come from another reviewer. 
```

Comment:
Replace with:
```
Approvers have the ability to merge a PR. An approver can indicate `/lgtm` or `/approve` to have a pull request merged, and all pull requests require at least one approver to provide their vote in order for the PR to be merged.
```
So:
> Approvers have the ability to merge a PR. An approver can indicate `/lgtm` or `/approve` to have a pull request merged, and all pull requests require at least one approver to provide their vote in order for the PR to be merged.

Reply:
still sounds confusing ... what doest a '/lgtm' from an approver mean then? It means a "tech LGTM"? It means the automated clicking of the "merge" button? 
What is the difference between '/approve' and '/lgtm' then? Equivalent?
Are we retiring the "Doc LGTM", "Tech LGTM" labels?

Reply:
@tengqm yeah - heading to retiring the DOC LGTM and Tech LGTM labels, instead aiming to utilize the informal hold when we want to lock something open until one or the other is accomodated, which puts us in the same process setup as most of the other kubernetes repos. The approver `/lgtm` is a bit of special-case, “big red button” kind of thing that will push through the otherwise required “separate /lgtm and /approve” setup that the automation uses. It’s not entirely obvious what will happen in those cases, so I struggled a bit with how to present it in a way that could be understood